R/coercion.r

```diff
@@ -608,3 +608,43 @@ setMethod("as.character", signature(x = "Duration"), function(x, ...){
 setMethod("as.character", signature(x = "Interval"), function(x, ...){
   format(x)
 })
+
+
+#' Change an object to a Date
+#'
+#' as_date changes \code{\link{POSIXt}}, numeric and character objects to
+#' \code{\link{Date}} objects.
+#'
+#' @param x a vector of \code{\link{POSIXt}}, numeric or character objects
+#' @param tzone a time zone name (default: time zone of the POSIXt object
+#'   \code{x}). See \code{\link{olson_time_zones}}.
+#' @param epoch a Date object, or something which can be coerced by
+#'   \code{as.Date(epoch, ...)} to such an object (default: the Unix epoch of 1
+#'   Jan 1970 as a \code{Date} object). Note that in this instance, \code{x} is
+#'   assumed to reflect the number of days since \code{eoch} at \code{"UTC"}.
+#' @param fmt a character string (default: \code{\%Y-\%m-\%d}). See
+#'   \code{\link{strptime}} for alternative format specifications.
+#' @param ... further arguments to be passed to specific methods (see above).
+#' @return a vector of \code{\link{Date}} objects corresponding to \code{x}.
```

Comment:
Could you please add a couple of words on the motivation. Why did we add this functions and how it compares to base as.Date?